build.gradle

```diff
@@ -77,13 +70,18 @@ ext.compatibilityVersion = '1.8'
 sourceCompatibility = compatibilityVersion
 targetCompatibility = compatibilityVersion
 
-jar {
-    manifest {
-        attributes 'Implementation-Title': 'Gradle Clover plugin',
-                   'Implementation-Version': version,
-                   'Built-By': System.getProperty("user.name"),
-                   'Built-Date': new Date(),
-                   'Built-JDK': System.getProperty("java.version")
+// Only add changing data to the manifest when publishing
+gradle.taskGraph.whenReady { graph ->
+    if (graph.allTasks.any { task -> task instanceof PublishToMavenLocal || task instanceof PublishToMavenRepository }) {
```

Comment:
What about use cases where Ivy is used? I use Ivy for artifacts in my use cases at work.

Reply:
So this is just in the gradle-clover-plugin build.  If you are publishing this plugin to Ivy, then yeah, we should probably address those as well.  I just didn't see any evidence of Ivy publishing configuration in the build (in build.gradle or gradle/publishing.gradle).

Reply:
Ah, you are right. Never mind then. My mistake.